bundle/API/Repository/TagsService.php

```diff
@@ -173,25 +173,40 @@ public function getTagSynonymCount(Tag $tag, array $languages = null, $useAlways
      * @param int $offset The start offset for paging
      * @param int $limit The number of content objects returned. If $limit = -1 all content objects starting at $offset are returned
      * @param bool $returnContentInfo
+     * @param array $sortClauses
```

Comment:
Can we use `\eZ\Publish\API\Repository\Values\Content\Query\SortClause[]` as a typehint here? Same goes for this change in `Core\TagsService` and `SignalSlot\TagsService` classes.

Reply:
Fixed in https://github.com/netgen/TagsBundle/pull/98/commits/05f7dad27c86fbfdc247c7f08f2966bc63b3a183.